ui/packages/catalog-ui-search/src/main/webapp/component/filter-builder/filter-builder.less

```diff
@@ -40,6 +40,20 @@
       margin-right: @minimumSpacing;
     }
 
+    > .filter-rearrange {
+      .grab-cursor();
+      // margin-left: @minimumSpacing;
+      margin-right: @minimumSpacing;
+      width: 0.75 * @minimumButtonSize;
+      //height: ~'calc(100% + @minimumSpacing)';
+      opacity: 0.25;
+    }
+
+    > .filter-rearrange:hover {
+      opacity: 0.50;
```

Comment:
Should we be using @minimumOpacity?

Reply:
I'll mess around with it some more, but we reached these current hover/non-hover opacity values after a good amount of iteration with @TeresaHudson